src/mutations/member.ts

```diff
@@ -19,18 +19,19 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
       // Snapshot the previous value
       const previousMember = queryClient.getQueryData(CURRENT_MEMBER_KEY);
```

Comment:
Thanks for your changes! Perhaps remove the logic for `previousMember` as we are not using it anymore?